demo/video_interpolation_demo.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+
+import cv2
+import mmcv
+import torch
+
+from mmedit.apis import init_model, video_interpolation_inference
+
+VIDEO_EXTENSIONS = ('.mp4', '.mov', '.avi')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Restoration demo')
+    parser.add_argument('config', help='test config file path')
+    parser.add_argument('checkpoint', help='checkpoint file')
+    parser.add_argument('input_dir', help='directory of the input video')
+    parser.add_argument('output_dir', help='directory of the output video')
+    parser.add_argument(
+        '--fps',
+        type=int,
+        default=50,
+        help='frames-per-second in the output video, which is needed when'
+        'the input is image sequence and want to get an output video.')
```

Comment:
frame rate of the output video, which is needed when the input is an image sequence and a video is desired as output.

Reply:
Another comment: Since in frame interpolation, we usually reduce the frame rate according to the number of frames we interpolate. Do you think it is better to use a multiplier (e.g. `0.5, 0.25`), and the ouptut frame rate equals to the this number times the original frame rate?